mautic/mautic
# Do not strip script and style tags in Froala

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | Y
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Required: )
#### Description:
htmlAllowedTags and htmlRemoveTags were contradicting each other, and script tags were being stripped within the Froala Editor. This issue has been tackled before but keep rearing its ugly head. Making the htmlRemoveTags array an empty array will fix this issue.

Finishes what was started in #6148

#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Add a <script> tag within the Froala WYSIWYG in Code View (save)
3. That tag should not get stripped on refresh.
